back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -6,6 +6,7 @@ class OmniauthCallbackController < ApplicationController
   skip_after_action :verify_authorized
 
   def create
+    # Rails.logger.warn("Auth extra raw info JSON: #{request.env['omniauth.auth'].extra['raw_info'].to_h.to_json}")
```

Comment:
I often need it for debugging. This would be better, but not 100% sure, so added it to BE CDM https://github.com/CitizenLabDotCo/citizenlab/pull/6981/files